pkg/dataloader/releaseloader/platforms.go

```diff
@@ -42,7 +42,15 @@ func (okd *OKDRelease) GetStreams() []string {
 }
 
 func (okd *OKDRelease) BuildReleaseStreams(releases []string) []string {
-	return buildReleaseStreams(releases, okd.GetStreams())
+	streams := okd.GetStreams()
+	releaseStreams := make([]string, 0, len(releases)*len(streams))
+	for _, release := range releases {
+		for _, stream := range streams {
+			releaseStream := strings.Replace(release, "-okd", ".0-0.", 1) + stream
```

Comment:
is the release name in the release table something that we have control over to define?
lgtm though.

Reply:
the release name comes from the `Releases` table, but it has to match the release name that comes from prow and it would likely be a huge hassle to change to anything but `X.X-okd`